modules/ROOT/pages/xml/xml-xpath.adoc

```diff
@@ -221,10 +221,10 @@ Returning to the <<othello_lines, Othello lines>> example, imagine that you want
 ----
 
 * The first argument is the XPath expression.
-* The second one is the input document, in this case, the message payload.
-* The third one is the context properties. Because this case does not require any, the function passes an empty object (`{}`).
+* The second argument is the input document, which, in this case, is the message payload.
+* The third argument is the context properties, which in this case are not required, so the function passes an empty object (`{}`).
```

Comment:
these should be parallel. Use "which, in this case," or "which in this case" in both 224 and 225.